scripts/kind-e2e/e2e.sh

```diff
@@ -442,6 +442,21 @@ elif [[ $deploy = helm ]]; then
     setup_cluster3_gateway
 fi
 
+free -h
```

Comment:
This requires the `procps-ng` package (in `Docker.dapper`).

Reply:
thanks